### ISSUE TITLE
License Checklist

### ISSUE THREAD

**Issue by jessie14** [UBC-MDS/Group28-CovidTracker-R#28]
We need to examine the license for our project and consider whether this is the choice we want to make, or whether you want to change the license. Let's have a discussion to debate and reason the license choice by opening issues in both Python and R repositories.

**Comment by SiqiTao** [comment]
license checklist issue is here

**Comment by SiqiTao** [comment]
Hi guys, as to our license, we are using MIT license (by default) right now. While there are other licenses to choose, such as GNU APACHE or BSD, I personally prefer keeping MIT as the license of our package. What do you guys think?

**Comment by SiqiTao** [comment]
Some reasons why I prefer MIT license:

- First of all, it is simple and we have used it before. Keeping using it can avoid unnecessary complications

- MIT license encourages open-source development. It would be easier and more accessible for other people to use our package or make suggestions.

- MIT license gives permissions to commercial use, which may be beneficial/helpful in the future.

**Comment by SiqiTao** [comment]
Although the simplicity of MIT license can potentially cause some problems (such as that it allow people to copy our package with little restriction, and it doesn't have a full-scale coverage of events), since we are still in the initial stage of building our packages and the packages are relatively simple and basic, I don't think it would be a problem that we need to concern anytime soon.

**Comment by jessie14** [comment]
I agree with the points above. The MIT license seems preferable for our package because it is simple to understand and it permits the use in commercial, distribution, modification, and private use, which covers all potential scenarios of the usage of our package. It's also widely used and accepted in the python package development community, so it will be familiar to most users of our package.
